addons/mail/static/src/models/thread/thread.js

```diff
@@ -919,6 +920,39 @@ function factory(dependencies) {
             this.update({ custom_channel_name: newName });
         }
 
+        /**
+         * Search locally after partners.
+         * @param {string} searchTerm the search term
+         * @param {integer} maxAmount the max amount of desired results
+         * @return [mail.partner[]]
+         */
+        searchLocalSuggestedPartners(searchTerm, maxAmount=8) {
+            const searchRegexp = new RegExp(_.str.escapeRegExp(webUtils.unaccent(searchTerm)), 'i');
+            const filterFunc = partner => partner && (
+                (partner.email && searchRegexp.test(partner.email)) ||
+                (partner.name && searchRegexp.test(webUtils.unaccent(partner.name)))
+            );
+            if (this.model === 'mail.channel') {
+                return this.members.filter(filterFunc);
+            }
+            const partners = [];
+            const partnerGroups = this._getOrderedPreferredPartnerMentionsGroups();
+            for (const partnerGroup of partnerGroups) {
+                if (partners.length >= maxAmount) {
+                    break;
+                }
+                for (const partner of partnerGroup.filter(filterFunc)) {
+                    if (!partners.find(p=>p.id === partner.id)) {
```

Comment:
`findFromIdentitying()`

Reply:
It's a simple js array, findFrom.. is not defined here